azure-client-authentication/src/main/java/com/microsoft/azure/credentials/RefreshTokenClient.java

```diff
@@ -32,7 +30,8 @@
     private final RefreshTokenService service;
 
     RefreshTokenClient(String baseUrl, Proxy proxy) {
-        OkHttpClient.Builder builder = new OkHttpClient.Builder().addInterceptor(new LoggingInterceptor(LogLevel.BODY_AND_HEADERS));
```

Comment:
Are we not logging body and headers any more?

Reply:
This needs work. LoggingInterceptor is missing because it turned into LoggingPolicy.